pom.xml

```diff
@@ -85,7 +92,7 @@
                             <java.util.logging.manager>
                                 org.jboss.logmanager.LogManager
                             </java.util.logging.manager>
-                            <maven.home>${maven.home}</maven.home>
+                            <maven.home>${env.MAVEN_HOME}</maven.home>
```

Comment:
Please, do not set the maven home in a pom.xml. The maven home is an environment variable that should be provided by the os system.